tests/wpt/mozilla/tests/mozilla/window_resizeTo_not_permitted.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Test that calling resizeTo() on normal window does not crash</title>
+<script>
+    window.resizeTo(-1, -1);
```

Comment:
The filename needs to end in `-crash.html` to integrate correctly with the test harness.